### ISSUE TITLE
Disable Trash

### ISSUE THREAD

**Issue by kajzersoze** [vifm/vifm#392]
I don't need it at all, how can I disable it ?

**Comment by durcheinandr** [comment]
If you can't find it in the documentation it's better to ask at the Vifm Q&A Site: https://q2a.vifm.info/

**Comment by kajzersoze** [comment]
I couldn't find it anywhere, even in that link from @xaizek there's no `set notrash`

Anyway, that solved it, thanks and sorry for bothering you with such a trivial and obvious solution to my stupid question, I didn't bother to look it up somewhere, went straight here to make some noise.

Won't happen again, deeply apologize for disturbance.

**Comment by xaizek** [comment]
Text at that link references description of [`'trash'`](https://vifm.info/vimdoc.shtml#vifm-%27trash%27) option in the second sentence. If you don't understand where `no` prefix comes from, you might want to read documentation on [`:set`](https://vifm.info/vimdoc.shtml#vifm-%3Aset) command.

And, yeah, too many apologies :) Q2A is just a better place for such questions, it's not like you did something wrong.

> I think it's quite easy to find in the documentation:

Correction: only if you know how options work in general.

**Comment by kajzersoze** [comment]
I did searched for it first, Q2A was the first place, then manual and docs, then issues here then google. For someone who just encountered `vifm` it might be little difficult to catch `no` prefix.

I dare you to link me anything which contains "set notrash" on the interwebs :)

Simple mention of this (how to disable trash) in `vifmrc` in the comments above `set trash` would be logical, if it's not already in the documentation, clearly it seems to say everything apart how to disable it.

Anyways, all's good, I had to ask, because if that can't be solved I'm not interested to continue exploring this fm any further.
